docs/drafts/developer.md

```diff
@@ -97,4 +93,103 @@ make manifests
 
 **NOTE:** Run `make help` for more information on all potential `make` targets.
 
-More information can be found via the [Kubebuilder Documentation](https://book.kubebuilder.io/introduction.html).
```

Comment:
Not sure if we should remove this link. May be keep it?

Reply:
I think it's sort of out of the blue as-is? Any ideas how to preface it to make it fit somehow?

Reply:
@joelanford advice was to ditch it and I think that makes sense, it's a tie-in to SDK's upstream, right? Seems out of place in a developer doc for operator-controller. wdyt?